_components/pages/XRPLUseCases.tsx

```diff
@@ -0,0 +1,217 @@
+    },
+  ];
+  const uses: Array<UseCaseCategoryInfo> = [
+    {
+      id: "micropayments",
+      title: "Micropayments",
+      description:
+        "Developers are using the XRP Ledger to build innovative products for gaming, content, and web monetization, among other applications where currency is at the center.",
+    },
+    {
+      id: "wallets",
+      title: "Cryptocurrency Wallets",
+      description:
+        "Use the Ledger to build digital wallets to store private and public passwords and interact with various blockchains to send and receive digital assets, including XRP.",
+    },
+    {
+      id: "exchanges",
+      title: "Exchanges",
+      description:
+        "Build sophisticated exchanges where users can invest and trade crypto and non-blockchain assets such as stocks, ETFs, and commodities.",
+    },
+    {
+      id: "stablecoins",
+      title: "Stablecoins",
+      description:
+        "Financial institutions can use Issued Currencies to issue stablecoins on the XRP Ledger. XRPL’s integrated decentralized exchange (DEX) allows neutral, counterparty-free digital assets to be seamlessly exchanged to and from “issued assets,” including stablecoins.",
+    },
+    {
+      id: "nft",
+      title: "NFTs",
+      description:
+        "XRPL allows issuance of IOUs which can represent a currency of any value, which can be extended to the issuance of non-fungible tokens (NFTs).",
+    },
+    {
+      id: "defi",
+      title: "DeFi",
+      description:
+        "Provide access to financial products and services online in a decentralized and borderless manner on XRPL, with decentralized smart contract protocols replacing the traditional role of financial institutions.",
+    },
+    {
+      id: "cbdc",
+      title: "CBDCs",
+      description:
+        "The CBDC Private Ledger provides Central Banks a secure, controlled, and flexible solution to issue and manage Central Bank Issued Digital Currencies (CBDCs).",
+    },
+  ];
+
+  return (
+    <div className="overflow-hidden page-uses landing-builtin-bgr">
+      <section className="py-26 text-center">
+        <div className="col-lg-5 mx-auto text-center">
+          <div className="d-flex flex-column-reverse">
+            <h1 className="mb-0">Powering Innovative Technology</h1>
+            <h6 className="eyebrow mb-3">XRPL Use Cases</h6>
+          </div>
+        </div>
+      </section>
+
+      <section className="container-new py-26">
+        <ul className="card-grid card-grid-3xN ls-none" id="use-case-card-grid">
+          {uses.map((use: UseCaseCategoryInfo) => {
+            const alt = `${use.title} icon`;
+            return (
+              <li className="col ls-none">
+                <img id={use.id} alt={alt} />
+                <h4 className="mt-3 mb-0 h5">{use.title}</h4>
+                <p className="mt-6-until-sm mt-3 mb-0">{use.description}</p>
```

Comment:
This is the element with white-on-white text in light mode. (See my comment on line 156 above for the suggested fix.)